cloud/aws/templates/aws_oidc/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "aws_region" {
 variable "civiform_image_repo" {
   type        = string
   description = "Public ECR repository with Civiform images"
-  default     = "public.ecr.aws/t1q6b4h2/universal-application-tool"
+  default     = "civiform/civiform"
```

Comment:
Is this correct?  If we are using the docker repo, then the description should change.

Reply:
I think it will technically work with both. Changed to Dockehub to avoid confusion.